logback.xml

```diff
@@ -16,6 +16,7 @@
   -->
 
 <configuration>
+    <property name="LOGS_DIR" value="/var/lib/tomcat/logs" />
```

Comment:
I don't think you need this (there is a default specified below if `LOGS_DIR` isn't set: `${LOGS_DIR:-./logs}`).

Also `/var/lib/tomcat` is specific to our Docker images (and if it is omitted from the file, it would be pulled from the System Properties instead, which is probably what we want by default, see http://logback.qos.ch/manual/configuration.html).

Reply:
Unfortunately the default (relative) value did not seem to work. I suppose this is because the current directory is not what we expect and `./logs` does not exist. The [way we used to do it](https://github.com/killbill/killbill-cloud/blob/work-for-release-0.21.x/ansible/templates/killbill/logback.xml.j2#L3) for the shared `logback.xml` was to specify the `LOGS_DIR`  var.

Reply:
I agree that `./logs` is a poor default (cannot remember where it comes from). Could we specify `${LOGS_DIR:-.}` instead (that way, it works out of the box)?

Reply:
I tried the `${LOGS_DIR:-.}`, but it does not work any better. I suggest we abandon this approach and use the `CATALINA_OPTS` to specify the `LOGS_DIR`. I'll submit another PR for this.